### PR TITLE
0.2.3 commits merging up into master

### PR DESCRIPTION
**Pull request by pc494**
These late addition to the 0.2.x series resolve a couple of issues. Specifically:

1) Provide support for the import convention used in the demos
2) Correct a sign error in the `.get_plot_data() `method of `OrientationRegion`
3) Avoid a round-off error by comparing to 1e-9 precision in the overriden `__gt__ `method or `OrientationRegion`

The 0.2.3 release will be built off a branch, these can then be merged (pending some small packaging changes).

### CODE REVIEW

**Comment by pc494** [comment]
@dnjohnstone 

Can we hold off merging this into master until I've packaged 0.2.3 please? If you could check you can use this code to run the demos that would be ideal. You should only need to change the syntax on the import to have the convention kwarg set to "Krakow_Hielscher" I think

**Comment by dnjohnstone** [comment]
> @dnjohnstone
> 
> Can we hold off merging this into master until I've packaged 0.2.3 please? If you could check you can use this code to run the demos that would be ideal. You should only need to change the syntax on the import to have the convention kwarg set to "Krakow_Hielscher" I think

Sure - do you want to flag the things you want merging or else just merge yourself in this instance?

**Comment by pc494** [comment]
If you can confirm the demo works on your machine I'll sort the rest (and merge my own PRs in this case)

File: orix/quaternion/rotation.py [hunk 1]
```diff
@@ -325,10 +326,33 @@ def from_euler(cls, euler, convention="bunge", direction="crystal2lab"):
...
+            )
+            qgamma = Quaternion(
+                np.stack((np.cos(gamma / 2), zero, zero, np.sin(gamma / 2)), axis=-1)
+            )
+            data = qalpha * qbeta * qgamma
+            rot = cls(data.data)
+            rot.improper = zero
+            return rot
+
         if convention != "bunge":
```
Comment (dnjohnstone) [inline orix/quaternion/rotation.py:356]: Not sure this is the right condition now that we have Krakow_Hielscher also?

Comment (pc494) [reply orix/quaternion/rotation.py:356]: Pushed to #58

**Pull request merged**